### DISCUSSION TITLE
How to disable Dependency Dashboard?

### DISCUSSION THREAD

**Discussion by hschoefer** [renovatebot/renovate#11330] [Request Help] [answered]
Since [26.0.0](https://github.com/renovatebot/renovate/commit/13532ecd3d5b3fd2ad4a4db912cdc860f378418a) Dependency Dashboard is enabled by default via base:config. In my context this configuration leads to a new Dependency Dashboard issue every execution. [Official documentation](https://docs.renovatebot.com/configuration-options/) says it should be disabled by default.

I*m not able to disable Dependency Dashboard in `config.js` with `dependencyDashboard: false`.

Any ideas?

**Comment by viceice** [comment, marked as answer]
extend from `:disableDependencyDashboard` preset or add `:dependencyDashboard` preset to `ignorePresets`

https://docs.renovatebot.com/presets-default/#disabledependencydashboard
https://docs.renovatebot.com/configuration-options/#ignorepresets

**Reply by rarkins** [reply]
If `dependencyDashboard: false` doesn't work then we should document that it must be `:disableDependencyDashboard`.

**Reply by hschoefer** [reply]
Dependency Dashboard is disabled, if I add `:disableDependencyDashboard` to the `renovate.json`.

To be clear: I added `dependencyDashboard: false` in the `config.js` for my renovate instance. Is this perhaps not the right place?

**Reply by viceice** [reply]
Yes and no, it's also a regression of #9415, breaking change in [`v25.0.0`](https://github.com/renovatebot/renovate/releases/tag/25.0.0)

If you set `dependencyDashboard: false` in `config.js` and extend `config:base` in `renovate.json` the preset will override the global option.
It normall should work setting  `dependencyDashboard: false` in  `renovate.json`, but because of #9415 you need package rules to override preset settings.

**Reply by rarkins** [reply]
Today presets in global config are treated as if they are repo config and are resolved *after* global config. It's because people had the expectation that repo config could use `ignorePresets` to override

**Reply by rarkins** [reply]
Related: https://github.com/renovatebot/renovate/issues/11207

**Comment by viceice** [comment]
Can you please provide more info about your config?
I like to understand why renovate always opens a new issue when it runs.
I Like to fix that, so you can use the dashboard as it's very informative about whats going on in renovate.

**Reply by hschoefer** [reply]
After your help I understand why only 2 of 4 repositories got this issue. `repo1_fake` and `repo2_fake` have a `renovate.json` in root path. `repo3_fake` and `repo4_fake` are without `renovate.json`.

`config.js` from my renovate instance.
```
module.exports = {
    platform: 'github',
    logLevel: 'debug',
    onboardingConfig: {
        extends: [
            'config:base',
            ':disableDependencyDashboard'
        ],
    },
    repositories: [
        'repo1_fake',
        'repo2_fake',
        'repo3_fake',
        'repo4_fake'
    ],
    renovateFork: true,
    gitAuthor: "renovate <renovate@example.com>",
    username: "renovate",
    onboarding: false,
    printConfig: true,
    requireConfig: false,
    prCreation: "immediate",
    dependencyDashboard: false
};
```
* My `config.js` with `dependencyDashboard` attribute. After your advice I removed it and use `:disableDependencyDashboard` in `renovate.json`
* I extended `onboardingConfig` with `:disableDependencyDashboard` with your advice for future use.

`renovate.json` example which now works.
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":disableDependencyDashboard"
  ]
}
```

**Reply by viceice** [reply]
Ok, you are using github, so  `username` should match the username of your renovate token, then renovate won't open new dashboard issues on each run. That should solve your root cause.
